ASSETS.md

```diff
@@ -39,6 +39,26 @@ requirement:
     version.min: 7.1
   filebeat:
     version.min: 7.2
+
+# The order of the items listed here is the order they show up in the package overview.
+screenshots:
+  # The src path is relative from inside the package. The full path will be generated by the server
+  # and exposed through the API.
+- src: /img/overview-logs.png
+  title: This shows the overview of the logs dashboard.
+  # The type does not have to be set explicitly if it's in the file extesions
+  # but the server will extract it an expose it through the API
```

Comment:
👍 Nice to see the API will include this. The UI might not need it now or every time, but it will make things much easier when it is needed.

Reply:
Potentially we can skip it in the first version until we have a consumer. But good to have it in the definition.